templates/vue/src/store/mutations.js

```diff
@@ -130,7 +130,6 @@ export function completeQuestion(state, { nodeId, questionId, answerType, answer
   if (state.userAnswers[nodeId].activity[questionId] === undefined) {
     state.userAnswers[nodeId].activity[questionId] = { answers: {} }
   }
-
```

Comment:
Again, no reason to remove empty line - this is especially more relevant if you have not touched the file otherwise. Touching files will always increase the chances of merge conflicts, so better to avoid touching them if not necessary and relevant to the PR.